cost/azure/reserved_instances/recommendations/azure_reserved_instance_recommendations.pt

```diff
@@ -0,0 +1,152 @@
+name "Azure Reserved Instances Recommendations"
+rs_pt_ver 20180301
+type "policy"
+short_description "A policy that sends email notifications when Azure RI Recommendations are identified. NOTE: These RI Purchase Recommendations are generated by Microsoft Azure. See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/azure/reserved_instances/recommendations) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
+long_description "Version: 1.0"
+severity "medium"
+category "Cost"
+
+permission "perm_read_creds" do
+  actions   "rs_cm.show_sensitive","rs_cm.index_sensitive"
+  resources "rs_cm.credentials"
+end
+
+parameter "param_enrollment" do
+  type "string"
+  label "Azure Enrollment ID"
+end
+
+parameter "param_days" do 
+  category "RI"
+  label "Look Back Period"
+  default "Last 7 Days"
+  allowed_values "Last 7 Days","Last 30 Days","Last 60 Days"
+  type "string"
+end 
+
+parameter "param_savings_threshold" do
+  category "RI"
+  label "Net Savings Threshold"
+  description "Specify the minimum net savings that should result in a recommendation"
+  default 100
+  type "number"
+end
+
+parameter "param_email" do
+  type "list"
+  label "Email addresses of the recipients you wish to notify"
```

Comment:
It was decided a while ago that all polices have the label and description.  In this case this the label is the description, and the label should be short "Email addresses to notify".